src/styles/partials/components/_modal.scss

```diff
@@ -0,0 +1,71 @@
+$default-modal-width: 640px;
+
+[role="alertdialog"],
+[role="dialog"] {
+  background-color: $pale-blue;
+  box-sizing: border-box;
+  color: $white;
+  min-height: 100vh;
+  padding: $padding-large $padding-largest;
+
+  * {
+    color: white;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  [role="alertdialog"],
+  [role="dialog"] {
+    position: absolute;
+    top: 2rem;
+    left: 50vw; /* move to the middle of the screen (assumes relative parent is the body/viewport) */
+    transform: translateX(
+      -50%
+    ); /* move backwards 50% of this element's width */
+    min-width: calc(
+      $default-modal-width - (15px * 2)
+    ); /* == breakpoint - left+right margin */
+    min-height: auto;
+    box-shadow: $modal-box-shadow;
+  }
+}
+
+.dialog_close_button {
+  float: right;
+  position: absolute;
+  top: 10px;
+  left: 92%;
+  height: 25px;
+}
+
+.dialog_close_button img {
+  border: 0;
+}
+
+/* native <dialog> element uses the ::backdrop pseudo-element */
+
+/* dialog::backdrop, */
+.dialog-backdrop {
+  display: none;
+  position: fixed;
+  overflow-y: auto;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+@media screen and (min-width: 640px) {
+  .dialog-backdrop {
```

Comment:
Should this be named `dg_dialog-backdrop` to be consistent with our class naming conventions?